pkg/controller/pod_test.go

```diff
@@ -155,7 +156,7 @@ func TestKleaner_DeletePod(t *testing.T) {
 	}
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
-			result := shouldDeletePod(tc.podSpec, tc.orphaned, tc.pending, tc.evicted, tc.successful, tc.failed)
+			result := shouldDeletePod(tc.podSpec, tc.orphaned, tc.pending, tc.evicted, tc.terminated, tc.successful, tc.failed)
```

Comment:
please add test scenario for every case that you're adding

Reply:
@lwolf  yes. I will be adding the test cases for each scenario and pushing one by one change for this PR